include/pfs/drp/stella/FiberTraceSet.h

```diff
@@ -118,6 +119,18 @@ class FiberTraceSet {
      */
     void sortTracesByXCenter();
 
+    /**
+     * @brief Extract spectra from an image
+     *
+     * We perform a simultaneous optimal extraction for all the traces using
+     * least-squares on each row.
+     *
+     * @param image : Image containing the spectra
+     * @param maskVal : Mask value to ignore.
+     * @return extracted spectra.
```

Comment:
Does this mean that `(val & maskVal) != 0` to ignore?  Or `|`?  Or something else?

Reply:
A pixel is ignored if `(value & maskVal) != 0`. This is how I've always described it, but since I write the code it makes sense to me. Do you have a better suggestion?

Reply:
I don't like the name, but if you say explicitly "ignore pixels where `(value & maskVal) != 0` in the comment it's OK.  I'd have called it something like `badBitMask`, but that's probably just me

Reply:
I like that name. In the future I'll try to break the decade-old habit of `maskVal`.